proto/src/main/proto/Rss.proto

```diff
@@ -62,6 +62,8 @@ message RequireBufferResponse {
   int64 requireBufferId = 1;
   StatusCode status = 2;
   string retMsg = 3;
+  // need split partitions
+  repeated int32 partitionIds = 4;
```

Comment:
why not marking the partitionIds name as the `needSplitPartitionIds`

Reply:
Done.